spec/system/support_users/feedback_spec.rb

```diff
@@ -96,6 +120,15 @@ def testid_for(time)
       expect(csv.second).to eq [other_feedback.created_at.to_s, "identified", "jobseeker", other_feedback.feedback_type, other_feedback.origin_path, other_feedback.email, other_feedback.occupation, other_feedback.rating, other_feedback.comment, other_feedback.category]
     end
 
+    it "allows user to download data for a specific date range" do
```

Comment:
Suggestion: I think the tests would be more solid if they explicitly test that there is no third row in the CSV (or any other way that the other feedback is not included) here. 
Same for the other test without a date range filter (it shouldn't include feedback older than the current month, right?).